lib/cocoapods/sandbox/pod_dir_cleaner.rb

```diff
@@ -48,14 +48,21 @@ def path_list
       # @return [Array<Strings>] The paths that can be deleted.
       #
       def clean_paths
-        cached_used = used_files
+        cached_used = used_files.map do |path|
+          path.downcase
+        end
         glob_options = File::FNM_DOTMATCH | File::FNM_CASEFOLD
-        files = Pathname.glob(root + '**/*', glob_options).map(&:to_s)
+        files = Pathname.glob(root + "**/*", glob_options).map(&:to_s)
```

Comment:
Why double quotes?

Reply:
VS Code extension converting to double, will revert back :)